tests/4D-Lung/4D-Lung.yaml

```
@@ -0,0 +1,20 @@
+# Config file for 4D-Lung for READII
+dataset_name: 4D-Lung 
+
+### CLINICAL VARIABLE INFORMATION ###
+# Event values should be in the order [Alive_value, Dead_value]
+outcome_variables:
+    time_label: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)